compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/BIRPackageSymbolEnter.java

```diff
@@ -380,6 +380,9 @@ private void defineImportPackage(DataInputStream dataInStream) throws IOExceptio
         BPackageSymbol importPackageSymbol = packageCache.getSymbol(importPkgID);
         //TODO: after bala_change try to not to add to scope, it's duplicated with 'imports'
         // Define the import package with the alias being the package name
+        if (importPackageSymbol == null) {
+            throw new BLangCompilerException("cannot resolve module " + importPkgID);
```

Comment:
If we are to introduce error reporting will need considerable changes. error reporting require package to be loaded atm (ie; for it to be in `BLangDiagnosticLog.packageCache`).  Hence, going with throwing exception